### PR TITLE
weevely 3.5 (new formula)

### PR DESCRIPTION
**Pull request by t94j0**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by commitay** [comment]
https://docs.brew.sh/Versions
> Versioned formulae submitted should be expected to be used by a large number of people.

> Upstream should have a release branch for the versioned formulae version and still make security updates for that version.

Please remove `weevely@3.4` and the alias.

**Comment by t94j0** [comment]
Updated to remove files requested.

File: Formula/weevely.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
+class Weevely < Formula
+  desc "Weaponized web shell"
+  homepage "https://github.com/epinna/weevely3"
+  url "https://github.com/epinna/weevely3/archive/v3.5.tar.gz"
+  sha256 "edb285469b2c8504797c1a35e30d728ddd88b56de37376c4464081c4d1cdbe9e"
+
+  depends_on "python@2"
```
Comment (fxcoudert) [inline Formula/weevely.rb]: Does it not work with macOS system python?

Comment (t94j0) [reply Formula/weevely.rb]: Yes, it does. I didn't know that it was recommended to use system python.

Comment (t94j0) [reply Formula/weevely.rb]: Fixed.

File: Formula/weevely.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  end
+
+  resource "readline" do
+    url "https://files.pythonhosted.org/packages/f4/01/2cf081af8d880b44939a5f1b446551a7f8d59eae414277fd0c303757ff1b/readline-6.2.4.1.tar.gz"
+    sha256 "e00f86e03dfe52e7d1502cec5c64070b32621c85509c0081a4cfa6a0294bd107"
+  end
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib64/python2.7/site-packages"
```
Comment (DomT4) [inline Formula/weevely.rb]: Where's the `lib64` coming from?

File: Formula/weevely.rb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    %w[PrettyTable Mako PyYAML python-dateutil PySocks readline].each do |r|
+      resource(r).stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+
+    (bin/"weevely").write <<~EOS
+      #!/usr/bin/env bash
+      cd #{libexec} && PYTHONPATH=#{ENV["PYTHONPATH"]} python2 weevely.py "$@"
```
Comment (DomT4) [inline Formula/weevely.rb]: `python2` doesn't exist by default on macOS. The system has `python` or `python2.7`

**Pull request closed without merging**